libjlm/include/jlm/opt/inlining.hpp

```diff
@@ -7,6 +7,8 @@
 #define JLM_OPT_INLINE_HPP
 
 #include <jlm/opt/optimization.hpp>
+#include <jive/rvsdg.hpp>
+#include <jlm/ir/operators/lambda.hpp>
```

Comment:
I believe that both headers are not necessary here. Forward declaring jive::input, jive::simple_node and lambda::node should be enough

Reply:
Seems like you assume you are talking to somebody that have a complete overview of the code.
If you meant that you think that jive/rvsdg.hpp is not needed, it has been removed.